src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -1454,41 +1454,211 @@ .maxstack  1
         // PROTOTYPE: Test unary operator- with `static IntPtr operator-(IntPtr)` defined on System.IntPtr. (Should be ignored for `nint`.)
         public static IEnumerable<object[]> UnaryOperatorsData()
         {
-            static void getArgs(ArrayBuilder<object[]> builder, string op, string opType, string expectedSymbol = null, DiagnosticDescription diagnostic = null)
+            static string getComplement(uint value)
```

Comment:
>getComplement [](start = 26, length = 13)

nit: this is not new to this PR, but local functions should be at the bottom of the method, so that the main code flow remains clear